db/migrate/20150612135546_create_host_status.rb

```diff
@@ -11,7 +11,29 @@ def up
     add_column :hosts, :global_status, :integer, :default => 0, :null => false
 
     Host.includes(:host_statuses, :last_report_object).find_each do |host|
-      host.refresh_statuses
+      begin
+        host.importing_facts = true
```

Comment:
the name is a bit confusing but basically it disables DHCP/TFTP orchestration that causes issue - https://github.com/theforeman/foreman/blob/develop/app/models/concerns/orchestration/dhcp.rb#L5